docs/ca/dev/preparation-for-going-live.md

```diff
@@ -23,191 +23,108 @@ We've divided the preparation into approximate timeframes, and you can adjust th
+| DAT-02 | Data     | Remove demo data                                  | Remove all the demo data from the environment. The project should only use the real data that will be used after the go-live. Remove all the demo data that comes with the Spryker repository, which includes demo and admin users. Demo users in a live shop pose a significant security risk for your project.
+| DAT-03 | Data     | Import real data                                  | Import real data to the production environment:<br>- Categories<br>- Product data, such as prices, offers, or attributes<br>- Customers, companies, users, merchants<br>- Taxes<br>- Discounts<br>- CMS pages: homepage, CSM blocks, terms & conditions<br>- Custom data<br>- Other data | Criteria: user journeys are running as expected. There're no blockers or unaddressed high-priority bugs. All the test cases and checklists prepared beforehand pass or don't block the release.
+| DAT-04 | Data     | Set up translations                               | Double-check that all the required languages are available.
+| DAT-05 | Data     | Define a glossary update strategy                 | Define the glossary update strategy. For example, exclude glossary updates during the normal deployment.
+| DAT-06 | Data     | Check emails                                      | Make sure that all emails are correct. Double-check that all the needed data and translations are available.
+| SEO-07 | SEO      | Optional: Set up redirects                       | Set up redirects.
 
 ## Go-live
-- Perform the end-to-end testing on production.
-- Disable email sending restrictions, if any.
-- Remove basic authentication from the frontend part and deploy the change.
-- Run the go-live communication plan.
-- Disable the destructive pipeline after the successful go-live process.
+
+| ID     | CATEGORY                                     | NAME                                         | DESCRIPTION                                                                    | NOTE                                                                         |
+|--------|----------------------------------------------|----------------------------------------------|--------------------------------------------------------------------------------|------------------------------------------------------------------------------|
+| TES-12 | Testing | Perform end-to-end testing on production | Perform smoke testing on production. Because its final the stage, test the main business flows and third-party integrations.
+| CLO-16 | Cloud    | Disable email sending restrictions           | Disable email sending restrictions.
+| CLO-17 | Cloud    | Remove basic auth                            | Remove basic authentication from the frontend part and deploy the change.
+| GEN-9  | General  | Run the go-live communication plan           | Run the go-live communication plan.
```

Comment:
Perhaps add more details in the description?

Reply:
Next iterations. For now lets keep as it was.